cmd/detectExecuteScan.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"fmt"
+	"sliceUtils github.com/SAP/jenkins-library/pkg/piperutils"
```

Comment:
Sorry, I think it's actually this:
```suggestion
	sliceUtils "github.com/SAP/jenkins-library/pkg/piperutils"
```